tensorflow/compiler/tests/unary_ops_test.py

```diff
@@ -95,8 +95,6 @@ def AssertAllEqual(self, result, expected, rtol, atol):
     """Tests that result and expeted are exactly equal."""
     self.assertAllEqual(result, expected)
 
-  @test_util.disable_mlir_bridge(
```

Comment:
This test has other failures as well. Keep this test disabled and change the description to "Handle complex element types in DiagPart op lowering"